bot.py

```diff
@@ -186,7 +189,7 @@ def __fetch_feed(self, feed_info):
                             newsdate = "no date"
 
                     # Update the database. If it's a new issue, post it to the channel
-                    is_new = self.__db.insert_news(feed_info[0], newstitle, newsurl, newsdate)
+                    is_new = self.__db.insert_news(feed_info[0], newstitle, newsitem.link, newsdate)
```

Comment:
The `newsurl` variable is still used when posting the link to IRC. By always storing the original URL in the database, toggling the short URLs option will avoid double posting previously seen feed items.